### ISSUE TITLE
Custom homepage from `file:///` URI is downloaded, not opened

### ISSUE THREAD

**Issue by machinedgod** [fanglingsu/vimb#715]
### Steps to reproduce
1. create a custom html file
2. configure vimb with a home-page parameter
4. attempt to navigate home with `gh`

### Expected behaviour
Custom homepage is opened.

### Actual behaviour
Homepage file is sent for download.

**Comment by fanglingsu** [comment]
@machinedgod Can you open hte home page if you open it via `:open file:///...`? If not you might try to use the file path direct as `:open /...`. For me both works.
If both does not work for you - this might be of the guessed mime type of the file. And it might not work in case you file name contains spaces.

**Comment by machinedgod** [comment]
It works both with `open` command with file URL and direct path - and now that I've tried setting it again, it works normally :exploding_head: 

Really sorry, really have no idea what changed! I've done few system updates and set up an external downloader (Persepolis). I haven't touched the homepage path _or_ the config line inside my config file. I have tried removing the download manager just for test, and it still works
:thinking:  :shrug: 

I'll close this. Thanks for the help.
